core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -225,6 +225,32 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
     waitForTopics(client, List(), topics)
   }
 
+  @Test
+  def testCreateExistedTopicsThrowTopicExistsException(): Unit = {
+    client = AdminClient.create(createConfig())
+    val topic = "mytopic"
+    val topics = Seq(topic)
+    val newTopics = Seq(
+      new NewTopic(topic, 1, 1.toShort),
+    )
+    client.createTopics(newTopics.asJava, new CreateTopicsOptions().validateOnly(true)).all.get()
+    waitForTopics(client, List(), topics)
+
+    client.createTopics(newTopics.asJava).all.get()
+    waitForTopics(client, topics, List())
+
+    val newTopicsWithInvalidRF = Seq(
+      new NewTopic(topic, 1, (servers.size + 1).toShort),
+    )
+    try {
+      client.createTopics(newTopicsWithInvalidRF.asJava, new CreateTopicsOptions().validateOnly(true)).all.get()
+      waitForTopics(client, topics, List())
+    } catch {
+      case e: ExecutionException =>
```

Comment:
We can replace try-catch with:
```
val e = val e = intercept[ExecutionException] {
      client.createTopics(newTopicsWithInvalidRF.asJava, new CreateTopicsOptions().validateOnly(true)).all.get()
}
```
That would also ensure we fail if the call doesn't throw an exception.